gettsim/data/e_st_abzuege.yaml

```diff
@@ -755,10 +787,10 @@ sauntmx:
       note: .
 sbhp100:
   name:
-    de: Pauschbeträge für behinderte Menschen, Hinterbliebene und Pflegepersonen (Grad der Behinderung 95% oder 100%)  (Höhe § 33b (1), (3) EStG)
+    de: Pauschbeträge für behinderte Menschen, Hinterbliebene und Pflegepersonen (Grad der Behinderung 95% oder 100%)
```

Comment:
name too long. can also be in a separate cleanup PR. also the following instances of sbhp...